packages/sanity/src/core/preview/documentPreviewStore.ts

```diff
@@ -132,6 +133,7 @@ export function createDocumentPreviewStore({
     id: string,
     apiConfig?: ApiConfig,
   ): Observable<string | undefined> {
+    // TODO: Account for version.
```

Comment:
Is this and the next TODO still valid? You are now receiving the `version` in options. 
Also, the availability seems to account now for version

Reply:
Great point, and I think you're correct. `observeDocumentTypeFromId` is called for each version of the document (published, draft, checked-out version), so the implementation itself doesn't need to deal with this. I'll remove the TODO.